test/helpers/helpers.go

```diff
@@ -150,6 +151,18 @@ func NewMachineConfigPool(name string, mcSelector, nodeSelector *metav1.LabelSel
 	}
 }
 
+// NewMachineConfigFile returns a file object created from arguments passed to it
+func NewMachineConfigFile(path, data string, mode int) *ign3types.File {
+
+	tempFile := ign3types.File{
+		Node: ign3types.Node{Path: path},
+		FileEmbedded1: ign3types.FileEmbedded1{Contents: ign3types.Resource{Source: StrToPtr(dataurl.EncodeBytes([]byte(data)))},
+			Mode: &mode}}
+
+	return &tempFile
+
+}
```

Comment:
I recently added a similar function which I think you could use: https://github.com/openshift/machine-config-operator/blob/master/pkg/daemon/update_test.go#L585

Unless do you need mode for some reason?

Reply:
I don't need mode, and it is practically the same function, but I'd have to go fish it out of daemon to use it. I can take the mode argument off of the one I put in helpers?